### PR TITLE
eas filters clean up TM-4036

### PR DESCRIPTION
**Pull request by SidelineCory24**
To-do
- [x] Standardize search/filter by field sizes (make the same size)
- [x] Remove search button
- [x] Remove red X from showing after entering a value
- [x] Always keep spyglass
- [x] Search on enter
- [x] Search on spyglass click
- [x] Remove Filter By heading 

image

### CODE REVIEW

**Comment by elizabeth-jimenez** [comment]
I know the ticket didn't explicitly point this out and i haven't pulled the code, so maybe it was already addressed - we want to make the name filters match the other filters in style. You see how the name ones have rounded corners? maybe also the size, but idk if that would look weird

**Review by M00staff — COMMENTED** [review]
the code looks good - I'm going to wait for @elizabeth-jimenez to pull it down and take a look at how its all rendering before approving though, as she'll be more familiar with the changes being asked for in this ticket

**Comment by SidelineCory24** [comment]
> I know the ticket didn't explicitly point this out and i haven't pulled the code, so maybe it was already addressed - we want to make the name filters match the other filters in style. You see how the name ones have rounded corners? maybe also the size, but idk if that would look weird

ah ok got it, the ticket made it seem like match the width, will update

**Comment by SidelineCory24** [comment]
styling updates to match below dropdowns
image

**Comment by elizabeth-jimenez** [comment]
image
let's adjust the padding between them

**Comment by elizabeth-jimenez** [comment]
I'm not seeing the spyglass after typing

**Comment by SidelineCory24** [comment]
> image let's adjust the padding between them

this is where the clear filters appears now, do we want to move it above the filters and between the toggle or above the toggle?

image

**Comment by elizabeth-jimenez** [comment]
> > image let's adjust the padding between them
> 
> this is where the clear filters appears now, do we want to move it above the filters and between the toggle or above the toggle?
> 
> image

our SSs are different - you are showing a margin/padding between one set of filters and the others - mine was showing margin/padding within the same set

**Comment by SidelineCory24** [comment]
> > > image let's adjust the padding between them
> > 
> > 
> > this is where the clear filters appears now, do we want to move it above the filters and between the toggle or above the toggle?
> > image
> 
> our SSs are different - you are showing a margin/padding between one set of filters and the others - mine was showing margin/padding within the same set

updated
image

File: src/sass/_employeeSearch.scss [hunk 1]
```diff
@@ -56,8 +57,10 @@
       }
     }
 
-    .search-submit-button{
-      margin: 0;
+    @media screen and (max-width: 1502px) {
```
Comment (elizabeth-jimenez) [inline src/sass/_employeeSearch.scss:60]: you'll want to remove the media query on L54

Comment (SidelineCory24) [reply src/sass/_employeeSearch.scss:54]: done

**Comment by elizabeth-jimenez** [comment]
image
still showing this at 1506 - remember to check media queries that may be affecting ur code

**Comment by SidelineCory24** [comment]
> image still showing this at 1506 - remember to check media queries that may be affecting ur code

not seeing at 1506 on my end, please let me know what I'm missing to replicate
image

**Review by elizabeth-jimenez — APPROVED** [review]
🐯

**Pull request merged**